### DISCUSSION TITLE
Volume not displaying when adjusting, desktop switcher not working with latest version

### DISCUSSION THREAD

**Discussion by FeliciaMenten** [valinet/ExplorerPatcher#2003] [General]
I'm using version 22621.1992.56.3 and it appears that two features that I use all the time have broken. 
1. The icon for displaying the desktops/desktop switcher has stopped working - clicking on it does nothing. 
2. When I adjust the volume using buttons on my keyboard, there used to be a popup that would appear to display the volume slider as I changed it.  That no longer happens. 

I can switch to the "Windows 11" taskbar style and they come back, so the problem seems to be with the Windows 10 taskbar style.   Which is why I use this app, so I can move the taskbar from the bottom to the right :)

thank you!

**Comment by pyrates999** [comment]
volume one is a duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1692

Work around until this gets fixed: install an application from the microsoft store called "modernflyouts".

desktop switcher not working is a duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1687

Work around: set to use windows 11 taskbar instead of windows 10 taskbar.

Please close this one.

**Comment by FeliciaMenten** [comment]
thank you!
